ocs_ci/utility/aws.py

```diff
@@ -457,6 +457,23 @@ def delete_volume(self, volume):
             "Delete response for volume: %s is: %s", volume.volume_id, delete_response
         )
 
+    def get_cluster_subnet_ids(self, cluster_name):
+        """
+        Get the cluster's subnet ids of existing cluster
+
+        Args:
+            cluster_name (str): Cluster name
+
+        Returns:
+            string of space separated subnet ids
+
+        """
+        subnets = self.ec2_client.describe_subnets(
+            Filters=[{"Name": "tag:Name", "Values": [f"{cluster_name}*"]}]
+        )
+        subnet_ids = [subnet["SubnetId"] for subnet in subnets["Subnets"]]
+        return ",".join(subnet_ids)
```

Comment:
Just small suggestion, I would return directly the list `subnet_ids` here - and join them with coma on the place, where you are using this method (it will be more reusable).